GoMFT update fails

### ✅ Have you read and understood the above guidelines?

yes

### 📜 What is the name of the script you are using?

GoMFT

### 📂 What was the exact command used to execute the script?

update

### 📝 Provide a clear and concise description of the issue.

when you try to update it give an error

### ⚙️ What settings are you using?

- [x] Default Settings
- [ ] Advanced Settings

### 🖥️ Which Linux distribution are you using?

_No response_

### 🔄 Steps to reproduce the issue.

   ______      __  _______________
  / ____/___  /  |/  / ____/_  __/
 / / __/ __ \/ /|_/ / /_    / /   
/ /_/ / /_/ / /  / / __/   / /    
\____/\____/_/  /_/_/     /_/     
                                  

  ✔️  Stopped GoMFT
⠦ Updating GoMFT to 0.1.17(!) templ version check: generator v0.3.856 is newer than templ version v0.3.833 found in go.mod file, consider running `go get -u github.com/a-h/templ` to upgrade
⠇ Updating GoMFT to 0.1.17(✓) Complete [ updates=67 duration=103.827227ms ]
⠏ Updating GoMFT to 0.1.17# runtime/cgo
cgo: C compiler "gcc" not found: exec: "gcc": executable file not found in $PATH

[ERROR] in line 49: exit code 0: while executing command $STD go build -o gomft

root@gomft:~# 

### ❌ Paste the full error output (if available).

   ______      __  _______________
  / ____/___  /  |/  / ____/_  __/
 / / __/ __ \/ /|_/ / /_    / /   
/ /_/ / /_/ / /  / / __/   / /    
\____/\____/_/  /_/_/     /_/     
                                  

  ✔️  Stopped GoMFT
⠦ Updating GoMFT to 0.1.17(!) templ version check: generator v0.3.856 is newer than templ version v0.3.833 found in go.mod file, consider running `go get -u github.com/a-h/templ` to upgrade
⠇ Updating GoMFT to 0.1.17(✓) Complete [ updates=67 duration=103.827227ms ]
⠏ Updating GoMFT to 0.1.17# runtime/cgo
cgo: C compiler "gcc" not found: exec: "gcc": executable file not found in $PATH

[ERROR] in line 49: exit code 0: while executing command $STD go build -o gomft

root@gomft:~# 

### 🖼️ Additional context (optional).

_No response_

--- a/ct/gomft.sh
+++ b/ct/gomft.sh
@@ -29,6 +29,10 @@ function update_script() {
     exit
   fi
   RELEASE=$(curl -s https://api.github.com/repos/StarFleetCPTN/GoMFT/releases/latest | grep "tag_name" | awk '{print substr($2, 3, length($2)-4) }')
+  if ! dpkg -l | grep -q "^ii.*build-essential"; then
+    $STD apt-get install -y build-essential
+  fi
+
   if [[ "${RELEASE}" != "$(cat /opt/${APP}_version.txt)" ]] || [[ ! -f /opt/${APP}_version.txt ]]; then
     msg_info "Stopping $APP"
     systemctl stop gomft

